specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/examples/GetAppTestRunsSearch.json

```diff
@@ -0,0 +1,140 @@
+                  "fileId": "92345678-1234-1111-1234-123456789011",
+                  "filename": "sample.jmx",
+                  "fileType": 2,
+                  "expireTime": "2021-12-05T16:43:46.072Z",
+                  "validationStatus": "VALIDATION_SUCCESS"
+                },
+                "userPropUrl": {
+                  "url": "https://dummyurl.com/userpropresource",
+                  "fileId": "92345678-1234-1111-1234-123456789013",
+                  "filename": "user.properties",
+                  "fileType": 1,
+                  "expireTime": "2021-12-05T16:43:46.072Z",
+                  "validationStatus": ""
+                },
+                "inputArtifactsZipFileurl": {
+                  "url": "https://dummyurl.com/inputartifactzipresource",
+                  "fileId": "92345678-1234-1111-1234-123456789012",
+                  "filename": "inputartifacts.zip",
+                  "fileType": 2,
+                  "expireTime": "2021-12-05T16:43:46.072Z",
+                  "validationStatus": ""
+                },
+                "additionalUrls": []
+              },
+              "outputArtifacts": {
+                "resultUrl": {
+                  "url": "https://dummyurl.com/dummyresourceresult",
+                  "fileId": "92345678-1234-1111-1234-123456789012",
+                  "filename": "sample.jmx",
+                  "fileType": 2,
+                  "expireTime": "2021-12-05T16:43:46.072Z",
+                  "validationStatus": "VALIDATION_SUCCESS"
+                },
+                "logsUrl": {
+                  "url": "https://dummyurl.com/logresource",
+                  "fileId": "abb846a7-63af-4d63-bb3d-a7aa4caff55c",
+                  "filename": "worker.log",
+                  "fileType": 2,
+                  "expireTime": "2022-05-16T09:14:46.0411793+00:00",
+                  "validationStatus": ""
+                }
+              }
+            },
+            "executedDateTime": "2021-12-05T16:43:48.125Z",
+            "vusers": 4,
+            "testRunStatistics": {
+              "Total": {
+                "transaction": "Total",
+                "sampleCount": 18,
+                "errorCount": 19,
+                "errorPct": 17,
+                "meanResTime": 13,
+                "medianResTime": 10,
+                "maxResTime": 16,
+                "minResTime": 18,
+                "pct1ResTime": 27,
+                "pct2ResTime": 20,
+                "pct3ResTime": 3,
+                "throughput": 5,
+                "receivedKBytesPerSec": 13,
+                "sentKBytesPerSec": 4
+              }
+            },
+            "createdDateTime": "2021-12-05T16:43:46.072Z",
+            "createdBy": "elon-musk@microsoft.com",
+            "lastModifiedDateTime": "2021-12-05T16:43:46.072Z",
+            "lastModifiedBy": "elon-musk@microsoft.com",
+            "portalUrl": "https://portal.azure.com/dummyresource",
```

Comment:
lets give some real portal url

Reply:
Is it necessary? portal.azure.com would remain the same. only the "dummyresource" part would change to another value